packages/legend-query-builder/src/stores/QueryBuilderValueSpecificationBuilder.ts

```diff
@@ -101,20 +119,30 @@ export const buildLambdaFunction = (
     ),
   );
 
-  // build getAll()
-  const getAllFunction = buildGetAllFunction(_class, Multiplicity.ONE);
-
   // build milestoning parameter(s) for getAll()
   const milestoningStereotype = getMilestoneTemporalStereotype(
     _class,
     queryBuilderState.graphManagerState.graph,
   );
-  if (milestoningStereotype) {
-    queryBuilderState.milestoningState
-      .getMilestoningImplementation(milestoningStereotype)
-      .buildGetAllParameters(getAllFunction);
+
+  if (milestoningStereotype && options?.isBuildingExecutionQuery === true) {
```

Comment:
Again i think we will break here for regular execution against milestone classes.